### DISCUSSION TITLE
How to setup on Oracle Ampere Instance

### DISCUSSION THREAD

**Discussion by blaazetech** [postalserver/postal#1580] [Help with using Postal]
Hi folks,

Am new to docker ecosystem
I have created an instance in oracle cloud with following specs
```
2 OCPUs
12GB RAM
50GB Boot Volume
Oracle-Linux-8.4-aarch64-2021.07.30-0
VM.Standard.A1.Flex
```
I have followed all the steps as mentioned in [docs.postalserver.io](https://docs.postalserver.io), but when i use `postal initialize` command, it throws error. 

Can anyone guide me please.

**Comment by willpower232** [comment]
what error did you get?

**Reply by blaazetech** [reply]
```
root@box2:~# postal initialize
Pulling web      ... done
Pulling smtp     ... done
Pulling worker   ... done
Pulling cron     ... done
Pulling requeuer ... done
Creating postal_runner_run ... done
standard_init_linux.go:228: exec user process caused: exec format error
ERROR: 1
```

**Reply by willpower232** [reply]
sorry I didn't realise ampere means arm processor so if you want to run Postal on it, you will need to make your own docker image.

**Reply by willpower232** [reply]
see also #1557
